src/Game.java

```diff
@@ -0,0 +1,67 @@
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static java.util.Map.Entry.comparingByValue;
+import static java.util.stream.Collectors.toMap;
+
+public class Game {
```

Comment:
interesting implementation of Game! Good job! But it has one big disadvantage. Think which one

Reply:
Only unused symbols can be used?